apps/total-typescript/src/pages/invoices/index.tsx

```diff
@@ -53,12 +53,13 @@ const Learn: React.FC<React.PropsWithChildren<{purchases: Purchase[]}>> = ({
                       <div>
                         <DocumentTextIcon
                           aria-hidden
-                          className="w-6 text-green-500"
+                          className="w-6 text-cyan-500"
                         />
                       </div>
                       <div>
                         <h2 className="text-lg font-semibold leading-tight">
-                          Testing Accessibility ({purchase.product.name}){' '}
+                          {process.env.NEXT_PUBLIC_SITE_TITLE} (
+                          {purchase.product.name}){' '}
```

Comment:
I mostly came here to update the background color and related styles, but noticed that we should be using the public site title here, so included that as well.